### DISCUSSION TITLE
object 語法問題

### DISCUSSION THREAD

**Discussion by s103071049** [Lidemy/forum#90] [其他問題]
不好意思，麻煩大家幫忙~
無法理解為甚麼 `console.log(`貨幣 : ${json[i].currencies.code}`)` 是 undefined ? 一定要加上 currencies[0] 才會印出 code 的值 ? 謝謝大家的幫忙~

[{"name":"Taiwan","topLevelDomain":[".tw"],"alpha2Code":"TW","alpha3Code":"TWN","callingCodes":["886"],"capital":"Taipei","altSpellings":["TW","Táiwān","Republic of China","中華民國","Zhōnghuá Mínguó"],"region":"Asia","subregion":"Eastern Asia","population":23503349,"latlng":[23.5,121.0],"demonym":"Taiwanese","area":36193.0,"gini":null,"timezones":["UTC+08:00"],"borders":[],"nativeName":"臺灣","numericCode":"158","currencies":[{"code":"TWD","name":"New Taiwan dollar","symbol":"$"}],"

``` js
const request = require('request')

const process = require('process')

const name = process.argv[2]

request(
  `https://restcountries.eu/rest/v2/name/${name}`,

  function (error, response, body) {
    const json = JSON.parse(body)
    const length = Object.keys(json).length

    for (let i = 0 ; i < length ; i++) {
        console.log('============')
        console.log(`國家 : ${json[i].name}`)
        console.log(`首都 : ${json[i].capital}`)
        console.log(`貨幣 : ${json[i].currencies[0].code}`)
        console.log(`國碼 : ${json[i].callingCodes}`)
    }
  }

)

```

**Comment by aszx87410** [comment]
因為 currencies 是個陣列

**Reply by s103071049** [reply]
對也>< 老師謝謝~
